BasicGates/BasicGates.ipynb

```diff
@@ -197,9 +197,9 @@
    "source": [
     "%kata T14_AmplitudeChange_Test\n",
     "\n",
-    "operation AmplitudeChange (q : Qubit, alpha : Double) : Unit is Adj {\n",
+    "operation AmplitudeChange (alpha : Double, q : Qubit) : Unit is Adj {\n",
```

Comment:
@tcNickolas: Sorry, I thought you meant for the one task; didn't want to make any further changes w/o asking.